pandas/tseries/frequencies.py

```diff
@@ -291,6 +292,11 @@ def to_offset(freqstr):
     if isinstance(freqstr, DateOffset):
         return freqstr
 
+    delta = None
+
+    if isinstance(freqstr, timedelta):
+        freqstr = tslib.Timedelta(freqstr)
```

Comment:
this test is not needed, you can just do `isinstance(freqstr, timedelta)` below, and then first convert to `Timedelta` (pandas.Timedelta will also be an instance of timedelta)